README.md

```diff
@@ -60,13 +60,15 @@ and use it to send `Commands`.
 For example, using SNS Client to publish a message to a topic looks like that:
 
 ```typescript
-import {PublishCommand, SNSClient} from '@aws-sdk/client-sns';
+import { PublishCommand, SNSClient } from "@aws-sdk/client-sns";
 
 const sns = new SNSClient({});
-const result = await sns.send(new PublishCommand({
-  TopicArn: 'arn:aws:sns:us-east-1:111111111111:MyTopic',
-  Message: 'My message',
-}));
+const result = await sns.send(
+  new PublishCommand({
+    TopicArn: "arn:aws:sns:us-east-1:111111111111:MyTopic",
+    Message: "My message",
+  }),
+);
```

Comment:
Please restore the original README formatting and only apply your changes.

Reply:
Will do